Adding section about forbidden errors (#5663)
DataDog/documentation: content/en/synthetics/troubleshooting/_index.md

{{< img src="synthetics/user-agent.gif" alt="Synthetics home page" responsive="true">}}

## API & Browser Tests

### Forbidden errors

> When first creating Synthetics tests, you might get `403 Forbidden` errors because of the `Sec-Datadog: Request sent by a Datadog Synthetics Browser Test (https://docs.datadoghq.com/synthetics/) - test_id: <TEST_ID>` header that is automatically being sent by Datadog.    

Reviewer: ```suggestion
When creating a Synthetics tests, you might get `403 Forbidden` errors at first. It's coming from the `Sec-Datadog: Request sent by a Datadog Synthetics Browser Test (https://docs.datadoghq.com/synthetics/) - test_id: <TEST_ID>` header that is automatically being sent by Datadog.    
```

> Make sure this header is not blacklisted by your servers.    

Reviewer: ```suggestion
Make sure this header is not blacklisted by your servers in order to remove this error.
```

> Additionally, you might also have to whitelist our [Synthetics IP ranges][2] to make sure our servers are allowed to send requests to your servers.

Reviewer: ```suggestion
Additionally, you might also have to whitelist [Datadog Synthetics IP ranges][2] to make sure Datadog servers are allowed to send requests to your infrastructure.
```
